.github/workflows/nightly.yml

```diff
@@ -14,20 +14,25 @@ jobs:
       uses: actions/checkout@master
       with:
         submodules: 'recursive'
+    - task: Npm@1
+      inputs:
+        command: 'install'
+        workingDir: 'Apps'
+      displayName: 'Install Babylon.js NPM packages'
```

Comment:
Why install npm packages at all? It seems like we don't need them.

Reply:
was just doing this right now to insure the directories existed. It doesn't seem like I can trigger this ci easily in my feature branch

Reply:
There is got to be a way to create directories without install npm packages. :-)

Reply:
this should now be fixed :)